README.md

```diff
@@ -460,7 +460,7 @@ debug-output-folder: $(az-output-folder)/_az_debug
 
 use-extension:
   "@autorest/python": "5.4.0"
-  "@autorest/clicommon": "0.6.1"
```

Comment:
Remember to change this as clicommon private release package first to see if everything works as expect. Then publish the clicommon package to npm.

Reply:
Hi @qiaozha , the clicommon is released and updated in this PR,  works fine in my side.